app/models/config.rb

```diff
@@ -10,7 +10,7 @@ class Config
   CONFIG_FIELDS = [
     :insurance, :external_pledge_source, :pledge_limit_help_text,
     :language, :resources_url, :fax_service, :referred_by,
-    :practical_support, :start_of_week
+    :practical_support, :show_practical_support, :start_of_week
```

Comment:
This is kinda particular so please forgive me, but showing practical support is the default behavior here, right? Would this make more sense as `hide_practical_support`, which I think infers that most of the time it should show up? Compare it with, for example, `if current_user.allowed_data_access?` in menu.html.erb -- that indicates that generally, the thing that's wrapping isn't the default behavior. Would love your thoughts on this.

Reply:
That's a reasonable objection - I'll do some more wordsmithing here.